curriculum/challenges/english/24-b1-english-for-developers/learn-common-phrasal-verbs-and-idioms/6854226946beb9af211d440c.md

```diff
@@ -0,0 +1,68 @@
+---
+id: 6854226946beb9af211d440c
+title: Task 43
+challengeType: 19
+dashedName: task-43
+---
+
+<!-- READING -->
+
+# --instructions--
+
+Read the text and answer the question below.
+
+# --questions--
+
+## --text--
+
+After talking to Anna, Brian comes up with the following first draft for the policies:
+
+**`Policy Update Draft`**
+
+`To align with our flexible work model, the policy no longer includes fixed in-office days. Employees can choose their work location, as long as they meet their responsibilities.`
+
+`We have added new communication guidelines for remote work. Team members should check in daily via our company platform and respond to messages within a reasonable time during work hours.`
+
+`The annual leave request system has been simplified. Employees will now complete a short online form that goes directly to their manager for quick approval.`
+
+`We also encourage all employees to take part in professional development. Courses, workshops, or learning new tools are all ways to grow skills. Managers will support team members in choosing useful learning paths.`
+
+`These updates will be reviewed by team leaders next week. After that, the final version will be shared with the whole company.`
+
+When did Brian create the first draft of the policy updates?
```

Comment:
For reading tasks, I only created questions based on the text between backticks. It's good to know we can also ask a question based on the introducing sentence, like this one, "After talking to Anna, Brian comes up with the following first draft for the policies:"